pysatSpaceWeather/tests/test_instruments.py

```diff
@@ -156,16 +157,39 @@ class TestSWInstrumentLogging(object):
 
     def setup_method(self):
         """Create a clean the testing setup."""
+        # Prepare for testing downloads
+        self.saved_path = pysat.params['data_dirs']
+        pysat.params.data['data_dirs'] = [pysatSpaceWeather.test_data_path]
```

Comment:
```suggestion
        pysat.params.data['data_dirs'] = [pysatSpaceWeather.test_data_path]
        pysat.params._set_data_dirs(path=pysatSpaceWeather.test_data_path, store=False)
```
recommend using the `store=False` option here.  If a user is running tests and doing something in parallel, their home directory will be rerouted to the test path while these tests are running.